### ISSUE TITLE
Node information not displayed for k3s kubernetes

### ISSUE THREAD

**Issue by mtseet** [unxsist/jet-pilot#45]
No node information is displayed for k3s kubernetes. Node is not shown as part of the artifacts listed on the left menu.
kubectl get nodes does show proper node information.

ubuntu 22.04
k3s kubernetes: rancher/k3s:v1.28.14-k3s1 in docker.

**Comment by unxsist** [comment]
I've checked this on my side with rancher k3s on Ubuntu 24.04 and Nodes are listed under the "Cluster" navigation group. It shows all nodes as well.

**Comment by mtseet** [comment]
Yes you are right. You can close the case. I did not have kubectl in my path that is why node information was not shown.
Thanks.

**Comment by unxsist** [comment]
Awesome, thanks for checking.
